webapp/src/main/webapp/WEB-INF/conf/news/portal-configuration.xml

```diff
@@ -35,5 +35,32 @@
         </object-param>
       </init-params>
     </component-plugin>
+    <component-plugin>
+      <name>new.portal.config.user.listener</name>
+      <set-method>initListener</set-method>
+      <type>org.exoplatform.portal.config.NewPortalConfigListener</type>
+      <description>this listener delete some predefined portal and templates configuration</description>
+      <init-params>
+        <object-param>
```

Comment:
component-plugin not needed, you can add this object-param group.configuration directly into the above new.portal.config.user.listener on the same level of portal.configuration